daprdocs/content/en/reference/components-reference/supported-pubsub/setup-aws-snssqs.md

```diff
@@ -9,7 +9,7 @@ aliases:
 
 ## Component format
 
-To set up AWS SNS/SQS pub/sub, create a component of type `pubsub.aws.snssqs`. See the [pub/sub broker component file]({{< ref setup-pubsub.md >}}) to learn how ConsumerID is automatically generated. Read the [How-to: Publish and Subscribe guide]({{< ref "howto-publish-subscribe.md#step-1-setup-the-pubsub-component" >}}) on how to create and apply a pub/sub configuration.
+To set up AWS SNS/SQS pub/sub, create a component of type `pubsub.aws.snssqs`. The AWS SNS/SQS component automatically generates the SNS topics and provisions SQS.
```

Comment:
I'd write so to emphasize:  `By default, the AWS SNS/SQS .....`. I'm a little concerned with the word "automatically". my proposal is therefore this:
By default, the AWS SNS/SQS component generates the SNS topics, provisions the SQS queues, and configures a subscription of the queues to the topics.
I also don't know if this is the right sentence to detail the fact to without a subscriber (i.e. only publisher that tries to publish a message exists) only the SNS topics are created, whereas if there is a subscriber (with dynamic or static subscription) sns, sqs and the subscription thereof would be generated

Reply:
@amimimor thanks for the review! I've updated - [how is this?](https://github.com/dapr/docs/pull/3611/files#diff-4dd30a3d9cc019d1b443b651b5a7e408d0426b684e8492f160bf10a950679bd3)

Reply:
LGTM!